dashboard/test/ui/runner.rb

```diff
@@ -641,7 +641,7 @@ def cucumber_arguments_for_browser(browser, options)
   arguments += skip_tag('@chrome_before_62') if browser['browserName'] != 'chrome' || browser['version'].to_i == 0 || browser['version'].to_i >= 62
   # browser version 0 implies the latest version.
   arguments += skip_tag('@no_older_chrome') if browser['browserName'] == 'chrome' && (browser['version'].to_i != 0 && browser['version'].to_i <= 67)
```

Comment:
worth cleaning up `@chrome_before_62` and `@no_older_chrome` at all, now that we are only running one version of Chrome? FWIW, I think the history behind Chrome44Win7 is that we had a test that started failing in latest chrome, but we found that it passed in older chrome, so this was our way of making sure that it kept running.

Reply:
@wjordan I will take a crack at removing these, unless you've started on this already.